app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::Base
+  default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
```

Comment:
unclear on the impact of this. Seems like it could have wide reaching impacts  - is it not better to use the ad-hoc basis rather than the global basis as defined in the [gems setup](https://github.com/DFE-Digital/govuk_design_system_formbuilder#setup-) here

Reply:
I thought so as well, however, during manual testing, nothing happened unless another form builder tries to call the govuk_design_system_formbuilder elements, i.e `= form.govuk_text_field`.

I will remove this anyway and call it on an ad-hoc basis as your comment.